cogs/game_controller.py

```diff
@@ -0,0 +1,75 @@
+import discord
+from discord.ext import commands
+from lib.game import DiscordGame
+import asyncio
+from typing import Any
+import textwrap
+
+REACTION_YES = "\U0001f44d"
+REACTION_NO = "\U0001f44e"
+REACTIONS = [REACTION_YES, REACTION_NO]
+
+
+class Game(commands.Cog):
+    def __init__(self, bot: Any) -> None:
+        self.bot = bot
+        self.db = self.bot.db
+
+    @commands.command()
+    @commands.cooldown(1, 30)
+    async def apply(self, ctx: commands.Context, target_member: discord.Member) -> None:
+        author_data = await self.db.get_user(ctx.author.id)
+        if author_data is None:
+            await ctx.send("あなたはユーザー登録されていません。")
+            return
+        target_member_data = await self.db.get_user(target_member.id)
+        if target_member_data is None:
+            await ctx.send("対戦申し込み先のユーザーがユーザー登録されていません。")
+            return
+
+        msg_text = f"""\
+        {target_member.mention}
+        {ctx.author.mention} に対決を申し込まれました。
+        受ける場合は :+1:
+        拒否する場合は :-1:
+        """
+        confirm_msg: discord.Message = await ctx.send(textwrap.dedent(msg_text))
+        for reaction_emoji in REACTIONS:
+            await confirm_msg.add_reaction(reaction_emoji)
+
+        def check(check_reaction: discord.Reaction, member: discord.Member) -> bool:
+            if not check_reaction.message == confirm_msg:
+                return False
+            if not str(check_reaction.emoji) in REACTIONS:
+                return False
+            if not member == target_member:
+                return False
```

Comment:
```suggestion
            if check_reaction.message.id != confirm_msg.id:
                return False
            if str(check_reaction.emoji) not in REACTIONS:
                return False
            if member != target_member:
                return False
```
messageはidで比較しないと動かなかったです。
あと、`not ... ==` や `not ... in` より `... !=` 、 `... not in`のがわかりやすいと思います。

Reply:
あ、ほんとだ、Messageには`__eq__`メソッドが定義されてないですね...
`!=`のほうがわかりやすい件についても了解しました、反映させていただきます！
細かい指摘をありがとうございます！